src/sherpa_ai/config/__init__.py

```diff
@@ -39,6 +39,7 @@
 PINECONE_NAMESPACE = environ.get("PINECONE_NAMESPACE", "ReadTheDocs")
 PINECONE_ENV = environ.get("PINECONE_ENV")
 PINECONE_INDEX = environ.get("PINECONE_INDEX")
+INDEX_NAME_FILE_STORAGE = environ.get("INDEX_NAME_FILE_STORAGE", "langchain-demo")
```

Comment:
I'm not sure if I understand the purpose of this environment variable. Maybe, add a description in .env-sample